app/views/administration/settings/index.html.haml

```diff
@@ -81,6 +82,35 @@
       .form-group
         .col-sm-offset-2.col-sm-10
           = button_tag t('save_changes'), class: 'btn btn-success'
+    .tab-pane#styles
+      = setting_row('Appearance', 'Skin Style', options: [['Blue','skin-blue'],['Blue On Light','skin-blue-light'],['Yellow','skin-yellow'],['Yellow On Light','skin-yellow-light'],['Green','skin-green'],['Green On Light','skin-green-light'],['Purple','skin-purple'],['Purple on Light','skin-purple-light'],['Red','skin-red'],['Red On Light','skin-red-light'],['Black','skin-black'],['Black On Light','skin-black-light']])
+      %div.btn-group.col-sm-10{ :id => 'site-colors', 'data-toggle' => 'buttons'}
+        %label{ :class => 'btn style-blue'}
```

Comment:
@cessien the color options should go in [config/locales/en/admin.yml](https://github.com/churchio/onebody/blob/master/config/locales/en/admin.yml)

You can put the names in the file like this:

```yaml
admin:
  settings:
    Appearance:
      Skin Style:
        options:
          skin-blue: Blue
          skin-yellow: Yellow
```

Then, in the view you can use them like this:

```haml
 = setting_row 'Appearance', 'Skin Style', options: t('admin.settings.Appearance.Skin Style.options').invert
```

I'm thinking we should name the setting "Theme" though. Sound ok?